<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>420</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>420</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Mp3 player</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icon/icon/play.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 127);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::GroupedDragging</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 127);</string>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="vSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="art">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135, 0, 0);</string>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>121</width>
      <height>10</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>10</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>361</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>305</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="playbutton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="baseSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopbutton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextbutton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prebutton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="shuffle">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mute">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>listWidget</zorder>
   <zorder>vSlider</zorder>
   <zorder>time</zorder>
   <zorder>art</zorder>
   <zorder>infoLabel</zorder>
   <zorder>dial</zorder>
   <zorder>progressBar</zorder>
   <zorder>status</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuControls">
    <property name="title">
     <string>Controls</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="actionShuffle"/>
    <addaction name="actionMute"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionActionAboutUs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuControls"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/search.png</normaloff>:/icon/icon/search.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/play.png</normaloff>:/icon/icon/play.png</iconset>
   </property>
   <property name="text">
    <string>play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/pause.png</normaloff>:/icon/icon/pause.png</iconset>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/stop.png</normaloff>:/icon/icon/stop.png</iconset>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/next.png</normaloff>:/icon/icon/next.png</iconset>
   </property>
   <property name="text">
    <string>next</string>
   </property>
   <property name="toolTip">
    <string>Next Song</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/pre.png</normaloff>:/icon/icon/pre.png</iconset>
   </property>
   <property name="text">
    <string>previous</string>
   </property>
   <property name="toolTip">
    <string>Previous Song</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
  <action name="actionActionAboutUs">
   <property name="text">
    <string>actionAboutUs</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>aboutQt</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionShuffle">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/unshuffle.png</normaloff>:/icon/icon/unshuffle.png</iconset>
   </property>
   <property name="text">
    <string>shuffle</string>
   </property>
   <property name="toolTip">
    <string>Shuffle songs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/mute.png</normaloff>:/icon/icon/mute.png</iconset>
   </property>
   <property name="text">
    <string>mute</string>
   </property>
   <property name="toolTip">
    <string>Mute</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
